### DISCUSSION TITLE
Allow Time fields in "Add field from calculation" transformations

### DISCUSSION THREAD

**Discussion by scottlepp** [grafana/grafana#38191] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
When choosing fields for "Reduce Row" and "Binary Calculation", add Time fields not just numeric.

**Why is this needed**:
So we can do calculations on time fields.

This works if I manually update the PanelJson and add the time fields, so it seems they should be available in the select.

**Comment by hugohaggmark** [comment]
Hi Scott, friendly reminder to add labels to all created issues so that the person on Issue Triage doesn't have to. Thank you.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
